### PR TITLE
Remove flag emoji suggestion

### PR DESCRIPTION
**Pull request by toastal**
Flags do not represent languages, they represent countries/nations. There's so many problematic reasons for this that it would just be better to not even start any convention. [ISO 639-3 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) are acceptable. You could change this to the 639-2 or 639-3 codes, but please *don't* use flags. Related to emoji, the Unicode consortium is no longer making any new flag emojis as they are so problematic and Unicode doesn't want to play with geopolitics.

Some additional resources::
* https://wplang.org/never-use-flags-language-selection/
* https://jkorpela.fi/flags.html

**Description**

**Your checklist for this pull request**

Make sure that this PR:
- [ ] Complies with our guidelines for contributions
- [ ] Has unit tests if possible.
- [ ] Has documentation updated

### CODE REVIEW

**Comment by SP2FET** [comment]
Hi, could you please create an issue and link it to this PR? It will be easier recognizable for us while doing community checks ;)

**Comment by toastal** [comment]
I don't plan to do anything additional.

> While the Pure's OS is open-source and can be viewed publically, Mudita doesn't support any modifications to it and will not help users in making any changes to the system.

Support has me to say this code is open source, there are no plans to actually give build instructions or instructions on how to flash the device with a custom build. Along with being on the source being on the closed-source, Microsoft-owned Git forge that is GitHub, I'm not sure if I want to lend more support to the project.

**Comment by pholat** [comment]
> I don't plan to do anything additional.
> 
> > While the Pure's OS is open-source and can be viewed publically, Mudita doesn't support any modifications to it and will not help users in making any changes to the system.
> 
> Support has me to say this code is open source, there are no plans to actually give build instructions or instructions on how to flash the device with a custom build. Along with being the source being on the closed-source, Microsoft-owned Git forge that is GitHub, I'm not sure if I want to lend more support to the project.

First of all, I have a feeling you felt offended - sorry for that, it was never our intention. 

Now let me straighten a few misconceptions:

1. Being open-source
Strictly speaking, means that the code is available openly and we do not limit access to it. You are more than free to modify it however you would love to too. It doesn't mean that we have to support any changes - it doesn't mean we won't or don't want to - it's just not our goal to help with code wishes, but to honestly and openly share what we have. The company has its own business goals and these don't have to align with requests all the time.

2. Building the package 
You are more than free to build your own package, if you would happen to actually modify the code you are actually required to run the simulator before contributing. 

3. Loading the software/package on the phone
I'm terribly sorry to hear that. AFAIR we plan to enable users to load their own modified software - it's just not our top priority at the moment, so it will take some time. Saying that - I'll double-check.

Actually, with a determined and skillful enough person, it would be fairly easy to either:
- load the package via jtag ( would require soldering some kynars and jlink, or some fun with pyocd)
- check how we load updates onto the phone and load your own package onto the phone. 

4. The platform we use doesn't really matter here.
We could host it on GitLab, self-hosted git, self-hosted Gerrit, or even pen drives if it matters. Github was just the most available. 

To sum up: please be so kind to help us out when asked, we really do our best and little kindness goes a long way.

**Comment by SP2FET** [comment]
@toastal  I'll throw a few cents to @pholat response - if you check closed issues/PRs from the community you'll see that we try to help as much as we can, but we can't fix all problems of user features - even if we want to, we don't have so much spare time.

If we can help then we do it, but we can't guarantee help from the begging to the end with community features development, that a contributor makes.
All that I asked for was just applying to the rules described here: https://github.com/mudita/MuditaOS/blob/master/CONTRIBUTING.md#feature-requests

We'll gladly take a look at this, but we don't want an unnecessary mess in our repository.
If you had any more questions feel free to ask :)

**Comment by SP2FET** [comment]
@toastal any response?

**Comment by Lefucjusz** [comment]
Closing due to lack of response.
